core/src/cluster_info.rs

```diff
@@ -1679,9 +1682,18 @@ impl ClusterInfo {
         let now = timestamp();
         let self_id = me.id();
 
+        //skip messages that are likely to be pushed
+        let min_filter_time = me.start_time + 10 * CRDS_GOSSIP_PULL_CRDS_TIMEOUT_MS;
+        let push_timer = if min_filter_time < now {
+            // reason for / 3 is to allow push_self which has a /2 timeout to propagate
```

Comment:
hmmm why `/3` vs `/2`? Shouldn't these numbers be based on something like the number of expected hops for a push message to propagate * how often a push message is propagated (`GOSSIP_SLEEP_MILLIS`)?